Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -682,6 +682,35 @@ void MainWindow::loadSettings()
 
     if(connect_ok)
         QTimer::singleShot(0, this, &MainWindow::slotConnectToLatest);
+
+#if defined(Q_OS_WINDOWS)
+    bool tolkLoaded = Tolk_IsLoaded();
+    if (!tolkLoaded)
+        Tolk_Load();
+
+    bool SRActive = Tolk_DetectScreenReader() != nullptr;
+
+    if (!tolkLoaded)
+        Tolk_Unload();

```

Comment:
I've made this correction. Previously Tolk would be unloaded since it's started by startTTS() a few lines up.